src/store/__mocks__/RelayStore.js

```diff
@@ -9,8 +9,8 @@
 
```

Comment:
tests should no longer be relying on RelayStore, so this mock can probably be simplified to exporting the actual module